packages/form-core/src/choice-group/ChoiceInputMixin.js

```diff
@@ -110,6 +110,17 @@ const ChoiceInputMixinImplementation = superclass =>
       if (changedProperties.has('modelValue')) {
         this.__syncCheckedToInputElement();
       }
+
```

Comment:
This enforces that you can't change radio `name` if it's different from the radio-group `name`. If you wanna change the name, it must be done through the parent.